test/build-test.ts

```diff
@@ -75,7 +76,7 @@ describe("build", () => {
       // renumber the hashes so they are sequential. This way we don’t have to
       // update the test snapshots whenever Framework’s client code changes. We
       // make an exception for minisearch.json because to test the content.
-      for (const path of findFiles(join(outputDir, "_observablehq"))) {
+      for (const path of sort(findFiles(join(outputDir, "_observablehq")), extname, (d) => d)) {
```

Comment:
I think the third argument `(d) => d` is being ignored here. Presumably we don’t need to pass any arguments to `sort` and should sort by the path?

Reply:
`findFiles` already sorts by path, by construction. However the annoying bit is the ordering of `inputs.xxxxx.css` vs `inputs.yyyyy.js`, which depends on both hashes. Sorting by `extname` ensures all the `css` are numbered before the `js`. But if we only sort by `extname`, then the css and the js are arbitrarily sorted within their group.

Reply:
Oh… I completely forgot that d3.sort accepts multiple accessors/comparators!